src/custom/components/swap/TradeSummary/index.tsx

```diff
@@ -101,6 +101,12 @@ export function RowReceivedAfterSlippage({
   )
   const isExactIn = trade.tradeType === TradeType.EXACT_INPUT
 
+  const [outAmount, outSymbol] = isExactIn
```

Comment:
nit: the name out here doesn't make sense for in amounts etc. i see what you're doing but maybe rethink the name to sth like `swapAmount` or similar